client/post-editor/editor-drawer/index.jsx

```diff
@@ -12,7 +12,7 @@ import Accordion from 'components/accordion';
 import AccordionSection from 'components/accordion/section';
 import Gridicon from 'components/gridicon';
 import CategoriesTagsAccordion from 'post-editor/editor-categories-tags/accordion';
-import EditorSharingAccordion from 'post-editor/editor-sharing/accordion';
+import EditorSharingAccordion from 'async-component!post-editor/editor-sharing/accordion';
```

Comment:
>Anything needed for `asyncRequire` and `AsyncLoad`. That way, I think we can split up changes to the build pipeline into more discrete pieces.

So, to be clear, for this instance we'd use `<AsyncLoad />` directly instead of the loader in this pull request, and defer the loader to a future one?

Just want to be sure we're including _something_ in this one. 😆